### PR TITLE
Liquibase rollback

### PR DESCRIPTION
**Pull request by anandjakhaniya**
Config parameter:
```
changeSetToRevertUntilTag: 
```
add it in section of `database:`
```
database:
  DBType: relational
  timeout: 4
  liquibaseLockThreshold: 60 #time in second
  changeSetToRevertUntilTag: db_version_1.4
```

### CODE REVIEW

File: backend/src/main/resources/liquibase/db-changelog-master.xml [hunk 1]
```diff
@@ -5,10 +5,36 @@
 
     <include file="src/main/resources/liquibase/db-changelog-1.0.xml"/>
 
+    <changeSet author="anandJ" id="before-demo-table">
```
Comment (ravishetye) [inline backend/src/main/resources/liquibase/db-changelog-master.xml]: these changes should be in db-changelog-2.0.xml  ? for POC to be closed to reality?

Comment (anandjakhaniya) [reply backend/src/main/resources/liquibase/db-changelog-master.xml]: it doesn't matter where the changeset tag is, but yes for the POC I can moved it to db-changelog-2.0.xml

**Comment by ravishetye** [comment]
@ad-47  can you create a PR with the actual logic and config flags? this PR can be in draft.

File: backend/src/main/java/ai/verta/modeldb/App.java [hunk 2]
```diff
@@ -210,6 +210,58 @@ public static void main(String[] args) {
...
+            configPassword,
+            changeSetToRevertUntilTag);
+
+        LOGGER.info("Liquibase validation stop");
+
+        boolean runLiquibaseSeparate =
+            Boolean.parseBoolean(System.getenv(ModelDBConstants.RUN_LIQUIBASE_SEPARATE));
+        if (runLiquibaseSeparate) {
+          return;
+        }
```
Comment (ravishetye) [inline backend/src/main/java/ai/verta/modeldb/App.java]: extract this to a function

**Review by ravishetye — APPROVED** [review]
minor comment looks good otherwise

File: backend/src/main/java/ai/verta/modeldb/App.java (hunk 2 above)
Comment (anandjakhaniya) [reply backend/src/main/java/ai/verta/modeldb/App.java]: Extracted

**Pull request merged**